gcf_data_mapper/read.py

```diff
@@ -0,0 +1,36 @@
+import csv
+import json
+from typing import Any, Optional
+
+import click
+
+
+def read_csv(file_path: str) -> list[str]:
+    """Reads a csv file and returns a list of values"""
+    with open(file_path, "r") as file:
+        csv_reader = csv.DictReader(file)
```

Comment:
This is fine for now - but performance wise this might be quite a costly way to search through CSV files.

Reply:
Ah for sure, this block will need to be updated anyway once we have a better idea of how the data will look in the csv's